front/src/routes/integration/all/zigbee2mqtt/device-page/DeviceTab.jsx

```diff
@@ -52,7 +52,11 @@ const DeviceTab = ({ children, ...props }) => (
               props.zigbee2mqttDevices.map((device, index) => (
                 <Zigbee2mqttBox {...props} device={device} deviceIndex={index} />
               ))}
-            {props.zigbee2mqttDevices && props.zigbee2mqttDevices.length === 0 && <EmptyState />}
+            {(!props.zigbee2mqttDevices || props.zigbee2mqttDevices.length === 0) && (
+              <EmptyDeviceList>
+                <Text id="integration.zigbee2mqtt.noDeviceFound" />
```

Comment:
The empty state was better before

Now: 
<img width="944" alt="Screenshot 2022-04-01 at 15 40 26" src="https://user-images.githubusercontent.com/7365207/161286150-67e332d3-e418-4406-8933-8346148e2bfa.png">

Before:
<img width="945" alt="Screenshot 2022-04-01 at 15 40 22" src="https://user-images.githubusercontent.com/7365207/161286160-41b2a09e-a583-4128-a19b-e873b08358d0.png">